plugins/source/azure/codegen0/internal/azparser/parser.go

```diff
@@ -0,0 +1,269 @@
+package azparser
+
+import (
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+// TODO: Skip NewKeyVault but also do this per package as otherwise I think we might be
+// skipping more than we want as clients are named the same across packages
+
+//
+
+// github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources
+var packagesToSkip = map[string]bool{
```

Comment:
Should these be added to the bash script? Otherwise they'll be added for the `go.mod` file and not be used